### DISCUSSION TITLE
Some (Hopefully) simple upgrade idea's

### DISCUSSION THREAD

**Discussion by TechTonic02** [Malcolm-Q/LC-LateGameUpgrades#508] [Ideas]
-First upgrade-
I've been in some situations where I thought a crouch speed increase upgrade would be nice, and thats exactly what im suggesting
-Second upgrade-
I personally would like to have a upgrade to reduce the uphill slope debuff (The movespeed reduction when walking up a slope)

Whether not these get added, I just wanted to let you know this mod is amazing, and thanks in advance

**Comment by WhiteSpike** [comment]
Doesn't the movement speed bonus provided by "Running Shoes" not contribute towards the crouch speed? I reckon crouching influencing how much this value contributes to this movement calculation so maybe you want to decrease how much the crouch state makes you go slower compared to walking?

This slope uphill, is only when you're going up a slope and this would just be a multiplier to decrease the effect of going uphill?

**Reply by TechTonic02** [reply]
Hey, thanks for the reply as to my suggestion:
If I'm understanding this right, then yes, that's how I imagined the crouching upgrade to be handled (if I'm getting this wrong, sorry lol)

Yeah that's all I want it to do, maybe it could have another effect but otherwise, that's all
